pkg/storage/chunk/storage/caching_fixtures.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/grafana/loki/pkg/storage/chunk/cache"
 	"github.com/grafana/loki/pkg/storage/chunk/gcp"
 	"github.com/grafana/loki/pkg/storage/chunk/testutils"
-	"github.com/grafana/loki/pkg/util/validation"
+	"github.com/grafana/loki/pkg/validation"
```

Comment:
The `github.com/grafana/loki/pkg/util/validation` points to a copy of the cortex validation package, which we should get clean up sometime.